endpoints/npm.ts

```diff
@@ -96,6 +100,15 @@ async function unpkg (topic, pkg, tag = 'latest') {
         color: 'green'
       }
     }
+    case 'esnext': {
+      const bundled = typeof meta.esnext === 'string'
+
+      return {
+        subject: 'esnext',
+        status: `${bundled ? '' : 'not '} bundled`,
```

Comment:
According to this https://2ality.com/2017/06/pkg-esnext.html#package-authors

`esnext` field is for "untranspiled source code", maybe we should change "bundled" to "included" or something.

Reply:
Mybe so. Not sure TBH.
As for me - `bundled` is more suitable here, but if you think `included` is better - i'll change it.

Reply:
I think we should be consistent with specification (if there is one 😅)